site/assets/js/snippets.js

```diff
@@ -75,28 +75,35 @@
   // -------------------------------
   // Alerts
   // -------------------------------
-  // Used in 'Show live alert' example in docs or StackBlitz
+  // Used in 'Show live toast' example in docs or StackBlitz
 
-  // Boosted mod: adapted innerHTML to have the icon and so added a parameter within `alert()` function
+  // Boosted mod: adapted innerHTML to have the icon and so added a parameter within alert() and a tooltip on btn-close
   // js-docs-start live-alert
   const alertPlaceholder = document.getElementById('liveAlertPlaceholder')
-  const alert = (message, type, typeVisuallyHidden) => {
+  function alert(message, type, typeVisuallyHidden) {
     const wrapper = document.createElement('div')
     wrapper.innerHTML = [
       `<div class="alert alert-${type} alert-dismissible" role="alert">`,
       '  <span class="alert-icon">',
       `    <span class="visually-hidden">${typeVisuallyHidden}</span>`,
       '  </span>',
-      '  <p>',
-      `    ${message}`,
-      '  </p>',
-      '  <button type="button" class="btn-close" data-bs-dismiss="alert">',
+      `  <p>${message}</p>`,
+      '  <button type="button" class="btn-close" data-bs-dismiss="alert" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close">',
       '    <span class="visually-hidden">Close</span>',
       '  </button>',
       '</div>'
     ].join('')
 
     alertPlaceholder.append(wrapper)
+
+    // Create btn-close's tooltip after innerHTML has been modified
+    const tooltip = new boosted.Tooltip(wrapper.querySelector('.btn-close'))
+    // Hide tooltip when clicking on live alert's btn-close
```

Comment:
The code is pretty obvious so I'd say either remove the comment, either make the following modification: "close button" or ".btn-close"